examples/demo/domain/pom.xml

```diff
@@ -27,29 +27,63 @@
 	<properties>
 	</properties>
 
-	<build>
-		<resources>
-			<resource>
-				<filtering>true</filtering>
-				<directory>src/main/resources</directory>
-				<includes>
-					<include>application.yml</include>
-				</includes>
-			</resource>
-			<resource>
-				<filtering>false</filtering>
-				<directory>src/main/resources</directory>
-			</resource>
-			<resource>
-				<filtering>false</filtering>
-				<directory>src/main/java</directory>
-				<includes>
-					<!-- we include all .java too, so that we can reference it from the descriptions -->
-					<include>**</include>
-				</includes>
-			</resource>
-		</resources>
-	</build>
+    <build>
+        <plugins>
+            <plugin>
```

Comment:
Maybe add a short explanatory comment, for the motivation of adding this plugin. (I guess some copy&past from this PR's description should suffice)